feat(template): enhanced the PR template to match Jira instead of the old github issues (#8)
antistatique/.github: .github/pull_request_template.md

### 💬 Describe the pull request
A clear and concise description of what the pull request is about.
### 💬 Describe the pull request/code changes
_A clear and concise description of what the pull request is about._

### 🗃️ Issues
This pull request is related to :
- #1
- #2
### 🎟️ Jira
_This pull request is related to the following Jira issue(s):_

### 🔢 To Review
Steps to review the PR:
1. Install '...'
2. Go to '...'
3. Click on '....'

> 4. Scroll down to '....'

Reviewer: Keep that or at least something similar; steps are too often skipped or unexplained.

Author: I agree, but I often delete all the steps to rewrite it. It's asked in the comments on the last lines

Reviewer 2: I agree that we could let that, even if we delete it, this example is a better incentive to write something,

Author: So @yannlugrin and @Yago can we agree to find maybe a better example or at least, less points ?

Reviewer 2: Write a proposal :)

Author: @yannlugrin Check the proposals on the readme of the PR.

Reviewer 3: I think having 2-3 multipoints on the "step to review" is more interesting, with 1 point, people will tend to add only 1 sentence (my point of view)

Author: @WengerK Personally, I still have to delete all the lines to rewrite them afterward bc placeholders like those don't fit my needs at all. Maybe the 1. 2. 3. empty is the best solution so in my case I could reuse it.

Reviewer 4: I agree with with them, 2-3 multipoints is a better incentive to write clear review steps. Plus it's really not that tedious to remove if needed.

Author: Ok, but can we agree to have something more fluent or appealing? Usually, I have to remove all the text from the steps. 

Have a look in the body of the PR for some ideas.

> - [ ] Update the "Unreleased" section of the `CHANGELOG.md`.  _(if applicable)_

Reviewer: ```suggestion
- [ ] Update the "Unreleased" section of the `CHANGELOG.md` with [chan](https://github.com/geut/chan/tree/main/packages/chan).  _(if applicable)_
```

> - [ ] Update the "NEXT RELEASE" section of the `CHANGELOG.md`. _(if applicable)_

Reviewer 2: ```suggestion
- [ ] Update the "Unreleased" section of the `CHANGELOG.md`.  _(if applicable)_
```

To match our current requirements

> [comment]: <> (Add the Jira issue)

Author: ```suggestion
[comment]: <> (Add the Jira ticket if needed/applicable)
```

### 💬 Describe the pull request/code changes
_A clear and concise description of what the pull request is about._

### 🎟️ Jira
_This pull request is related to the following Jira issue(s):_

### 🔢 To Review
_Steps to review the PR:_

> 1. 👷 🏗 🚧

Reviewer 5: ```suggestion
1. 
2. 
```

### 📝 Does it need an entry in the CHANGELOG?
 - [ ] Update the "NEXT RELEASE" section of the `CHANGELOG.md`

### 📸 Screenshots
_If applicable, add screenshots to help explain the task._

### 📦 Ready to merge
- [ ] The PR is covered by some tests. _(if applicable)_

[comment]: <> (Add steps to review the PR if needed/applicable)

> [comment]: <> (Add others conditiosn to allow the PR to be merged if needed/applicable)

Reviewer 5: ```suggestion
[comment]: <> (Add others conditions to allow the PR to be merged if needed/applicable)
```

### 💬 Describe the pull request/code changes
_A clear and concise description of what the pull request is about._

### 🎟️ Jira
_This pull request is related to the following Jira issue(s):_

### 🔢 To Review
_Steps to review the PR:_
1. Add some details for the reviewer here.

### 🔖 Does it need an entry in the CHANGELOG?
 - [ ] Update the "NEXT RELEASE" section of the `CHANGELOG.md`

### 📸 Screenshots
_If applicable, add screenshots to help explain the task._

### 📦 Ready to merge

> - [ ] The reviewer approves the pull request.

Reviewer 5: We already have this information with GitHub. What is the reason to duplicate it?

- [ ] The PR is covered by some tests. _(if applicable)_

> - [ ] Merges conflicts are resolved. _(if applicable)_

Reviewer 5: Same here, GitHub UI already give the information that there is a conflict

Author: Make sens, can remove
